sdks/python/apache_beam/io/gcp/experimental/spannerio.py

```diff
@@ -523,6 +585,42 @@ class _ReadFromPartitionFn(DoFn):
   """
   def __init__(self, spanner_configuration):
     self._spanner_configuration = spanner_configuration
+    self.base_labels = {
+        monitoring_infos.SERVICE_LABEL: 'Spanner',
+        monitoring_infos.METHOD_LABEL: 'Read',
+        monitoring_infos.SPANNER_PROJECT_ID: self._spanner_configuration.
```

Comment:
Please use this style, avoid breaking the self._spanner_configuration.project into mutliple lines
        monitoring_infos.SPANNER_PROJECT_ID: (
            self._spanner_configuration.project),

Reply:
Done